site2/website-next/sidebars.json

```diff
@@ -81,6 +85,17 @@
         "tiered-storage-azure",
         "tiered-storage-aliyun"
       ]
+    },
+    {
+      "type": "category",
+      "label": "Transactions",
+      "items": [
+        "txn-why",
```

Comment:
I think it might be more appropriate to put What are transactions as the first one, and then why transactions,.....